relayer/crates/starknet-chain-components/src/impls/types/config.rs

```diff
@@ -27,6 +27,7 @@ pub struct StarknetChainConfig {
 pub struct StarknetContractAddresses {
     pub ibc_core: Option<StarknetAddress>,
     pub ibc_client: Option<StarknetAddress>,
+    pub ibc_ics20: Option<StarknetAddress>,
```

Comment:
so, I should add this field in the `relayer.toml.example` file?

Reply:
If I'm not mistaken, this is currently only used by `build_ibc_token_transfer_message` to create a transfer. So it's shouldn't be an issue if it is not configured